cce/docker-compose.yml

```diff
@@ -31,6 +32,10 @@ services:
       BEAM_PROXY_URL: http://beam-proxy:8081
       RETRY_COUNT: ${FOCUS_RETRY_COUNT}
       EPSILON: 0.28
+      QUERIES_TO_CACHE: '/queries_to_cache.conf'
+      ENDPOINT_TYPE: ${FOCUS_ENDPOINT_TYPE:-blaze}
+    volumes:
+      - /srv/docker/bridgehead/cce/queries_to_cache.conf:/queries_to_cache.conf
```

Comment:
Make read-only?

Secondly, please know: If the file on the left doesn't exist, it will create an empty directory called queries_to_cache.conf.

Reply:
With #282 now merged, this is resolved.